### PR TITLE
Merge TestStatus and Readme

### PR DESCRIPTION
**Pull request by sampoearas**
Tested on 11.0 SP12, SP15

### CODE REVIEW

**Review by EliJDonahue — CHANGES_REQUESTED** [review]
* Built and tested on 11 **SP9**, not SP7. 
* Include the FF issue and lack of Chrome support in the release notes table entry for v1.0.1.

**Comment by sampoearas** [comment]
Readme Updated: FF issue moved. SP listing updated.

File: README.md [hunk 1]
```diff
@@ -3,18 +3,21 @@
...
 
-See [TESTSTATUS file](./TESTSTATUS.md) for latest testing information.
+Release | Notes
+--------|--------
+[v1.0.1](https://github.com/ArasLabs/dashboard-customizations/releases/tag/v1.0.1) | Tested 11.0 SP12, SP15. Tested on Edge, IE, ~~Firefox 60 ESR, Chrome.~~ **Package works in IE, Edge. [Has axis error in Firefox (Issue).](https://github.com/ArasLabs/dashboard-customizations/issues/1) DOES NOT WORK in Chrome.**
+[v1.0.0](https://github.com/ArasLabs/dashboard-customizations/releases/tag/v1.0.0) | First release. Tested on Internet Explorer 11, Firefox 38 ESR, Chrome. 
 
-#### Built Using:
-Aras 11.0 SP9
+> Though built and tested using Aras 11.0 SP9, this project should function in older releases of Aras 11.0 and Aras 10.0.
```
Comment (EliJDonahue) [inline README.md]: This line can be removed. Now we have the supported versions table to convey this info.
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -3,18 +3,21 @@
 This project demonstrates how to further customize dashboard charts in addition to adding custom font CSS to the chart labels.
 To add custom font CSS to a chart please follow the Change Font of Dashboard document in the documents folder.
 
-## Project Details
+## History
 
-See [TESTSTATUS file](./TESTSTATUS.md) for latest testing information.
+Release | Notes
+--------|--------
+[v1.0.1](https://github.com/ArasLabs/dashboard-customizations/releases/tag/v1.0.1) | Tested 11.0 SP12, SP15. Tested on Edge, IE, ~~Firefox 60 ESR, Chrome.~~ **Package works in IE, Edge. [Has axis error in Firefox (Issue).](https://github.com/ArasLabs/dashboard-customizations/issues/1) DOES NOT WORK in Chrome.**
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
[v1.0.1](https://github.com/ArasLabs/dashboard-customizations/releases/tag/v1.0.1) | Tested 11.0 SP12, SP15. Package works in IE, Edge. [Has axis error in Firefox (Issue).](https://github.com/ArasLabs/dashboard-customizations/issues/1) Chrome not supported at this time.
```

**Review by EliJDonahue — CHANGES_REQUESTED** [review]

**Comment by sampoearas** [comment]
changes accepted. Please merge.

**Review by EliJDonahue — APPROVED** [review]

**Pull request merged**